corehq/apps/reports/sqlreport.py

```diff
@@ -345,15 +345,11 @@ def format_rows(self, data, keys=None, group_by=None):
                 formatted_row = self._format.format_row(row)
                 if self.filter_row(row_key, formatted_row):
                     yield row_key, formatted_row
-        elif group_by:
+        else:
             for key, row in data.items():
                 formatted_row = self._format.format_row(row)
                 if self.filter_row(key, formatted_row):
                     yield key, formatted_row
-        else:
-            formatted_row = self._format.format_row(data)
-            if self.filter_row(None, formatted_row):
-                yield None, formatted_row
 
     def filter_row(self, key, row):
```

Comment:
nit: I don't see anywhere that creates a DataFormatter with a row filter so maybe we can just remove that completely? (maybe separate PR)

Reply:
Did we stop using it? 
https://github.com/dimagi/commcare-hq/pull/871/files#diff-10981343807562b1516015f8cff1d936R233

Reply:
I'm not sure it was ever used - I don't see any current usages.

Reply:
Cool, will take it up as a followup PR